src/soilbiogeochem/SoilBiogeochemCarbonFluxType.F90

```diff
@@ -349,7 +345,7 @@ subroutine InitHistory(this, bounds, carbon_type)
 
         this%decomp_cpools_leached_col(begc:endc,:) = spval
         this%decomp_cpools_transport_tendency_col(begc:endc,:,:) = spval
-        do k = 1, ndecomp_pools
+        do k = 1, ndecomp_pools  ! none from CWD
            if ( .not. decomp_cascade_con%is_cwd(k) ) then
```

Comment:
This if-statement pertains to leaching and there's a corresponding if-statement in `SoilBiogeochemNitrogenFluxType.F90`.

I did not remove these. Should I?

Reply:
...and I would like confirmation that I removed the correct if-statements in `SoilBiogeochemCarbonFluxType.F90` and `SoilBiogeochemNitrogenFluxType.F90`.

Reply:
One more `if (.not. is_cwd)` that I did NOT remove:

https://github.com/slevisconsulting/ctsm/blob/219c6c19e603b29fc74a80e66ee54c1e57ca0a4a/src/soilbiogeochem/SoilBiogeochemLittVertTranspMod.F90#L283

This one pertains to the handling of the spin-up. Again let's discuss whether it should be removed.